website/docusaurus.config.js

```diff
@@ -36,27 +36,16 @@ const config = {
           path: "../nitro/docs/",
           sidebarPath: require.resolve("./sidebars.js"),
           routeBasePath: "/",
-          editUrl: function(s) {
+          editUrl: function (s) {
             return (
-              "https://github.com/OffchainLabs/nitro/tree/docs/docs/" +
+              "https://github.com/OffchainLabs/nitro/edit/master/docs/" +
               s.docPath
             );
           }
         }
       })
     ]
   ],
-  plugins: [
```

Comment:
This broke sdk doc

Reply:
Can you point me to the docs directory for the sdk or how to run it correctly? I got an error running docusaurus without this change saying it can't find `/docs` in the sdk :(

Reply:
https://developer.offchainlabs.com/sdk

Reply:
Did you pull both submodules?

Reply:
Yes I pulled both. I will look into the sdk part tmr and reply here.

Reply:
right you actually need to run `yarn build` inside `./website/` to generate the sdk docs

Reply:
Thank you! I rebased it now. Didn't realise it required cd-ing into the sdk/ to generate docs.
Should we add a README that includes the build instructions on a new PR?

Reply:
https://github.com/OffchainLabs/nitro-docs/blob/b2775f6f313ead3adc9269ee959749a7a78a4419/website/package.json#L8

we can have the start script do similar thing as the build script e.g.
```
"start": "cd ../arbitrum-sdk && yarn && yarn generate_docs && cd ../website && docusaurus start",
```